test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -747,6 +755,80 @@ def __are_requirements_matching(param_dic=None, test=None):
             if not eff_tc_req['no_comm_on_mom']:
                 return False
 
+    def check_and_report_hardware_status(self):
+        """
+        function used to get hardware report
+        every 5 minutes
+        """
+        du = DshUtils()
+        self.hardware_report_timer = Timer(
+            300, self.check_and_report_hardware_status)
+        self.hardware_report_timer.start()
+        systems = list(self.param_dict['servers'])
+        systems.extend(self.param_dict['moms'])
+        systems.extend(self.param_dict['comms'])
+        systems = list(set(systems))
+        for hostname in systems:
+            hr = SystemInfo()
+            hr.get_system_info(hostname)
+            # monitors disk
+            used_disk_percent = getattr(hr,
+                                        'system_disk_used_percent', None)
+            if used_disk_percent is None:
+                _msg = hostname
+                _msg += ": unable to get disk info"
+                self.logger.error(_msg)
+                self.hardware_report_timer.cancel()
+                raise PbsHardwareMonitorError(msg=_msg)
+            elif 70 < used_disk_percent < 95:
+                _msg = hostname + ": used disk is "
+                _msg += str(used_disk_percent) + "%"
+                _msg += "Please clean the disk"
```

Comment:
can we make the message better - something like - "disk usage is at str(used_disk_percent)%, disk cleanup is recommended."

Reply:
this comment is not addressed completely.